docs/src/noncommutative.md

```diff
@@ -96,11 +96,11 @@ julia> Q, (x, y) = QuotientRing(G, std(I))
 ### WeylAlgebra
 
 ```julia
-function WeylAlgebra(R::Union{Ring, Field}, s::Vector{String};
+function WeylAlgebra(R::Union{Ring, Field}, s::Union{Vector{String}, Vector{Symbol}};
                      ordering = :degrevlex, ordering2::Symbol = :comp1min,
                      cached::Bool = true, degree_bound::Int = 0)
 
-function WeylAlgebra(R::Union{Ring, Field}, s::Matrix{String};
+function WeylAlgebra(R::Union{Ring, Field}, s::Union{Matrix{String}, Vector{Symbol}};
```

Comment:
Do you mean this?
```suggestion
function WeylAlgebra(R::Union{Ring, Field}, s::Union{Matrix{String}, Matrix{Symbol}};
```

Reply:
Huh, yes, but weird that the test code still passes. Something funny is going on.

Reply:
Oops. Didn't have Singular.jl dev`d.